setuptools_rust/__init__.py

```diff
@@ -8,6 +8,8 @@
 from .tomlgen import tomlgen_rust, find_rust_extensions
 from .utils import Binding, Strip
 
+__version__ = "0.11.6.dev1"
```

Comment:
To make it easier to do releases, can we use `setuptools_scm` version hard-coding?

e.g. in `pyproject.toml`:

```
[tool.setuptools_scm]
write_to = "pkg/version.py"
```

and then refer to the generated file here?

Reply:
Nice trick! I was not aware of this feature. Please check out the update to my PR.

Reply:
Thanks! I will test this quickly 👍